wsmaster/che-core-api-factory-bitbucket-server/src/test/java/org/eclipse/che/api/factory/server/bitbucket/HttpBitbucketServerApiClientTest.java

```diff
@@ -325,17 +331,17 @@ public void shouldGetOauth2Token()
         new HttpBitbucketServerApiClient(
             wireMockServer.url("/"), new NoopOAuthAuthenticator(), oAuthAPI, apiEndpoint);
     stubFor(
-        get(urlEqualTo("/rest/api/1.0/users/user"))
+        get(urlEqualTo("/rest/api/1.0/users/ksmster"))
             .withHeader(HttpHeaders.AUTHORIZATION, equalTo("Bearer token"))
             .willReturn(
                 aResponse()
                     .withHeader("Content-Type", "application/json; charset=utf-8")
                     .withBodyFile("bitbucket/rest/api/1.0/users/ksmster/response.json")));
 
     // when
-    bitbucketServer.getUser("user", null);
+    bitbucketServer.getUser(Optional.empty());
```

Comment:
@vinokurig would it be possible to have a `getUser` method without parameters?

Reply:
That's a good point, done